pkgs/development/python-modules/unicodeit/default.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  fetchpatch,
+  pytestCheckHook,
+  runCommand,
+
+  setuptools,
+  unicodeit,
+}:
+buildPythonPackage rec {
+  pname = "unicodeit";
+  version = "0.7.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "svenkreiss";
+    repo = "unicodeit";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-NeR3fGDbOzwyq85Sep9KuUiARCvefN6l5xcb8D/ntHE=";
+  };
+
+  patches = [
+    (fetchpatch {
+      # Defines a CLI entry point, so `setuptools` generates an `unicodeit` executable
+      url = "https://github.com/svenkreiss/unicodeit/pull/79.patch";
```

Comment:
As this pull request is stil open and might change, please reference a commit instead of the pull request